recipe/meta.yaml

```diff
@@ -29,10 +29,10 @@ requirements:
     - python
     - pip
     - numpy
-    - cython 0.29.33
+    - cython 0.29
     - setuptools
     - wheel
-    - packaging 22.0
+    - packaging 23
   run:
     - python
     - ipython >=2.0.0
```

Comment:
numpy entry is here twice - should leave in the one that pins to our numpy? also ipython seems like it would be in run_constrained. There are also a bunch of run_constrained entries that aren't included yet. https://github.com/yt-project/yt/blob/8a6dca3c554aebb7ccfc5a390ce0bd1148588fec/setup.cfg#L80

Reply:
to keep the recipe as close to what is on main, I would rather keep the changes to a minimum.
This could be corrected when updating to the next version.

Reply:
alright, fair enough